### PR TITLE
[codemod] markDynamoStrictTest batch 20

### PR DESCRIPTION
**Pull request by zou3519**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #117609
* __->__ #117606
* #117605
* #117604

[codemod] markDynamoStrictTest test_tensorexpr_pybind
[codemod] markDynamoStrictTest test_tensorexpr
[codemod] markDynamoStrictTest test_jit_llga_fuser
[codemod] markDynamoStrictTest test_jit_fuser_te

### CODE REVIEW

File: torch/testing/_internal/dynamo_test_failures.py [hunk 1]
```diff
@@ -7750,6 +7754,13 @@
     "TestProfiler.test_profiler_disable_fwd_bwd_link",
     "TestProfiler.test_profiler_metadata",
     "TestSerializeCustomClass.test_custom_class",
+    "TestTensorExprPyBind.test_kernel_with_transpose",
+    "TestTensorExprPyBind.test_kernel_with_custom_lowering",
+    "TestTensorExprPyBind.test_kernel_with_expand",
+    "TestTensorExprPyBind.test_kernel_with_permute",
+    "TestTensorExprPyBind.test_kernel_with_scalar_inputs",
+    "TestTensorExprPyBind.test_kernel_with_t",
+    "TestTensorExprPyBind.test_kernel_with_tensor_inputs",
```
Comment (bdhirsh) [inline torch/testing/_internal/dynamo_test_failures.py:7763]: dynamo'ing TensorExpr (NNC?) code, uh oh 😛

Comment (bdhirsh) [reply torch/testing/_internal/dynamo_test_failures.py:7763]: Just curious, why skip and not fail - flaky?

**Review by bdhirsh — APPROVED** [review]

File: torch/testing/_internal/dynamo_test_failures.py (hunk 1 above)
Comment (zou3519) [reply torch/testing/_internal/dynamo_test_failures.py:7762]: flaky: it failed locally for me, but passes in CI

**Comment by huydhn** [comment]
@zou3519 Some tests are failing on dynamo after this batch https://hud.pytorch.org/pytorch/pytorch/commit/308e154af5fd6388f49eabe631e7b78ca3ac9c39, just want to double check with you if fixes are coming.

**Comment by zou3519** [comment]
re-landed in another PR

**Pull request closed without merging**